nemo/utils/model_utils.py

```diff
@@ -636,3 +652,36 @@ def ckpt_to_dir(filepath: Union[str, Path]) -> Path:
     checkpoint_dir = filepath.with_name(filepath.stem)
 
     return checkpoint_dir
+
+
+def save_artifacts(model, output_dir: str, use_abspath: bool = False) -> None:
```

Comment:
Do we need this or can we use the existing implementation? I.e. the save/restore connector?

Reply:
Should be able to use register_artifact for this

Reply:
I took time to revisit this. This helper actually just copies artifacts from a source Nemo model (tar or dir) to a folder with quantized weights.

I would prefer using the helper because of two main reasons:
1. Most importantly the quantized model is actually a directory produced with AMMO export step here: https://github.com/NVIDIA/NeMo/blob/b56ff60381b80d0add4456297dab0fb52b30cf1e/nemo/export/quantize/quantizer.py#L184-L191 as opposed to a Nemo model offering `register_artifact` method.
2. Artifacts saved with `register_artifact` are prefixed with a MD5 hash. On the other hand, utils in Nemo Inference container typically assume hardcoded "plain" paths like `tokenizer.model` instead of  `449ae6fd76d84842bf152e4ae4701764_tokenizer.model` (for example). So I would need to perform and extra operation to remove this prefix somehwere.

Defining the `save_artifacts` helper gives me the flexibility I need. Are you OK with this?